site/_posts/2022-09-09-what-is-webauthn-why-do-you-care.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: blog-post
+title: What is WebAuthn and why do you care?
+description: WebAuthn is a new standard which secures user accounts and improves user experience. What could be better?
+author: Dan Moore
+image: blogs/sso-explanation/how-sso-works.png 
+category: blog
+tags: topic-single-sign-on
+excerpt_separator: "<!--more-->"
+---
+
+WebAuthn is a new way for people to authenticate themselves to web applications. It's a widely supported standard years in the making. All major browsers work with it, which makes it easy for developers to incorporate WebAuthn into websites.
+
+WebAuthn allows users to leverage the power of biometric, device and other authentication methods via a simple, consistent, browser-native user experience.
+
+<!--more-->
+
+Am I overstating things? Is WebAuthn really all that?
+
+Read on and decide for yourself.
+
+## What is WebAuthn?
+
+WebAuthn is a way for users to authenticate to applications running in the browser. Just like a username and password, WebAuthn provides credentials, but it identifies people with a different method. WebAuthn requires a special piece of hardware or software called an "authenticator".
+
+WebAuthn removes the need for an application or user store to save off private, valuable data such as a hashed password. Instead, the application stores a public key. The corresponding private key is held securely, and in a distributed fashion, by each user's authenticator.
```

Comment:
What is meant by "distributed fashion" here? Just that it's held on the end device rather than centrally by the application server? To me this sounds like private keys are distributed and available across all of a user's devices, which is not generally the case. (Apple is working on syncing them securely via iCloud or something)